sqlitecluster/SQLiteNode.cpp

```diff
@@ -235,6 +237,7 @@ void SQLiteNode::_sendOutstandingTransactions() {
         _sendToAllPeers(commit, true); // subscribed only
         _lastSentTransactionID = id;
     }
+    SINFO("Sent all " << transactionCount << " outstanding transactions.");
```

Comment:
I added it for some reason, but it may have been related to troubleshooting the other master flip flop issues. Removing for now, if we want it sometime later, we'll worry about how much additional volume it creates at that time.